packages/experimental/Button/src/CompoundButton/CompoundButton.types.ts

```diff
@@ -0,0 +1,55 @@
+import { ViewProps } from 'react-native';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { ButtonSlotProps, ButtonTokens, ButtonProps } from '../Button.types';
+import { FontTokens } from '@fluentui-react-native/tokens';
+
+export const compoundButtonName = 'CompoundButton';
+
+/**
+ * A baseline set of color plates.
+ */
+// export type ColorTokens = Partial<{
+//   background: string;
+//   contentColor: string;
+//   secondaryContentColor: string;
+//   linkColor: string;
+//   iconColor: string;
+//   borderColor: string;
+//   dividerColor: string;
+//   focusColor: string;
+//   focusInnerColor: string;
+//   opacity: string;
+// }>;
+
+// export type ColorTokenStates = Partial<{
+//   hovered: ColorTokens;
+//   pressed: ColorTokens;
+//   disabled: ColorTokens;
+//   checked: ColorTokens;
+//   checkedHovered: ColorTokens;
+//   checkedPressed: ColorTokens;
+// }>;
+
+// export type ColorTokenSet = ColorTokens & ColorTokenStates;
+
+export interface CompoundButtonTokens extends ButtonTokens {
+  secondaryContentTokens?: FontTokens & { color?: string };
+}
+
+export interface CompoundButtonProps extends ButtonProps {
+  /**
+   * Second line of text that describes the action this button takes.
+   */
+  secondaryContent?: string;
```

Comment:
Should this maybe be called caption?

Reply:
I also wonder if this could be implemented as recomposing Button, with replacing the content slot and the render function. Right now that might be a bit tricky but worth considering.

Reply:
I went with secondaryContent because that's what the [converged web button](https://github.com/microsoft/fluentui/blob/master/packages/react-button/src/components/CompoundButton/CompoundButton.types.ts) calls it.

Reply:
I see.  Are you suggesting I might take the contentContainer (View around content and secondaryContent) and put that in the  content slot of the base Button?

Reply:
Effectively yes, though I don't know whether it is a net gain.